.circleci/config.yml

```diff
@@ -35,48 +35,52 @@ jobs:
           key: repo-{{ .Environment.CIRCLE_SHA1 }}
           paths:
             - ~/repo
-  test-1:
+  test-contracts-ganache:
     docker:
       - image: circleci/node:9
     working_directory: ~/repo
     steps:
       - restore_cache:
           keys:
             - repo-{{ .Environment.CIRCLE_SHA1 }}
-      - run: yarn wsrun test:circleci @0xproject/contract-wrappers
-      - run: yarn wsrun test:circleci @0xproject/sol-compiler
-      - save_cache:
-          key: coverage-contract-wrappers-{{ .Environment.CIRCLE_SHA1 }}
-          paths:
-            - ~/repo/packages/contract-wrappers/coverage/lcov.info
-      - save_cache:
-          key: coverage-sol-compiler-{{ .Environment.CIRCLE_SHA1 }}
-          paths:
-            - ~/repo/packages/sol-compiler/coverage/lcov.info
-  test-contracts:
+      - run: yarn wsrun test:circleci contracts
+  test-contracts-geth:
     docker:
       - image: circleci/node:9
+      - image: albrow/0x-devnet
```

Comment:
Can you add an Asana task to move this image to the 0xproject docker account?

Reply:
So does this image contain both the Geth node and the node env. where the tests will run?

Reply:
So it took me a while to figure this out because it's not the most well-documented feature. The first image listed here is the "primary image" which will run all of our tests. Any additional images listed will be spun up independently by CircleCI (but in the same network).